CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- **BREAKING:** Removed support for Node v12 in favor of v14 ([#137](https://github.com/MetaMask/eth-json-rpc-middleware/pull/137))
```

Comment:
We can save updating the changelog / bumping the version to the end (the version probably needs to be 5.0.0 anyway).

Reply:
Just to be clear are you suggesting we leave the above addition to [Unreleased] out at this point?

Reply:
Removing the version bump is important because our release process relies on that not having been done yet.

This changelog entry is fine to leave here, but it's of limited help. These get auto-generated by our release process.The auto-generated entries aren't formatted as nicely, they need to be manually pruned and cleaned up, but it's enough that it's probably not worth your time doing this going forward.

Reply:
Yeah so just to summarize: leave the changelog entry, but drop the version bump.

Reply:
done